proto/documents/v1alpha/documents.proto

```diff
@@ -168,6 +172,11 @@ message ListPublicationsRequest {
 
   // Optional. Value from next_page_token obtains from a previous response.
   string page_token = 2;
+
+  // Optional. Whether to return trusted publications only
+  // or including all publications regardless of the trusted state
+  // By default, it returns all publications (trusted_publications_only = false)
+  bool trusted_publications_only = 3;
```

Comment:
Maybe call it the same as in GetPublication? `trusted_only` instead of `trusted_publications_only`. Also, I'm thinking whether `ignore_untrusted` could be a better name?

Reply:
I don't really like double negations, is a variable says `noSomething` and is set to false, My brain heats up a little bit trying to understand what the hell the user wants